assets/queries/cloudFormation/stack_retention_disabled/query.rego

```diff
@@ -37,10 +39,10 @@ CxPolicy[result] {
 	resource := input.document[i].Resources[name]
 	resource.Type == "AWS::CloudFormation::StackSet"
 
-	object.get(resource.Properties, "AutoDeployment", "undefined") != "undefined"
+	common_lib.valid_key(resource.Properties, "AutoDeployment")
```

Comment:
`common_lib.valid_key(resource.Properties, "AutoDeployment")` is unnecessary. Same for lines 45 and 81